### ISSUE TITLE
Support relative path in front-end

### ISSUE THREAD

**Issue by IamBusy** [ARCANEDEV/LogViewer#209]
- LogViewer Version: 4.4.2
- Laravel Version: 5.5
- PHP Version: 7.0

### Description:
A nginx is used as a reverse-proxy on my server. It will redirect requests which have a specific pattern like `http://my.domain.com/mypartern/` to my laravel project. So I can visit the `log-viewer` by `http://my.domain.com/mypartern/log-viewer`. However, when I click any buttons or links on the page, it will automatically redirect to `http://my.domain.com/target`, which should be  `http://my.domain.com/mypartern/target`. 

So I wonder how to handle this situation. Is there some configure item that I can set to cover it? 

Thanks.

**Comment by arcanedev-maroc** [comment]
I don't think this is related to `log-viewer` package.

However, you can add your `mypattern` prefix [here](https://github.com/ARCANEDEV/LogViewer/blob/master/config/log-viewer.php#L54) for `the log-viewer` routes.

**Comment by IamBusy** [comment]
Thanks for your reply. I tried to modify the prefix, which likes `mypattern/log-viewer`, but it doesn't solve the problem. If I set the prefix, the visiting url become to `http://my.domain.com/mypartern/mypartern/log-viewer`, which is unexpected. 

The radical reason is the front-end templates use absolute path as  the redirect links which like:
```html
<a href="{{ route('log-viewer::logs.list') }}">
       <i class="fa fa-archive"></i> Logs
</a>
```

It will be rendered as:

```
<a href="http://my.domain.com/log-viewer">
       <i class="fa fa-archive"></i> Logs
</a>
```

So I think it's a good idea to use relative path.

**Comment by IamBusy** [comment]
If there is no reverse-proxy server, it's all right.

This is part of my `nginx.conf`:

```
    server_name  my.domain.com; 

    location /myparttern/ { 
        proxy_pass http://log-viewer-project/;
        proxy_set_header   Host    $host; 
        proxy_set_header   X-Real-IP   $remote_addr; 
        proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for; 
    }
```

If I visit `http://log-viewer-project/log-viewer` directly, it's ok. No problem exists.
But if I visit the web sites by proxy server `http://my.domain.com/myparttern/log-viewer`, and click `Log` button, it will redirect to `http://my.domain.com/log-viewer/logs`, which will not be handled. Could you understand what's the situation I describe?

**Comment by arcanedev-maroc** [comment]
Yes, i understand your issue but this is not related to the `package`. 

It's related to **Laravel** with **reverse-proxy**.

Check this one: https://stackoverflow.com/questions/29912997/laravel-routes-behind-reverse-proxy

**Comment by yotewach** [comment]
I've an issue too
